tests/brain/python/compartmentReport.py

```diff
@@ -75,32 +75,36 @@ def test_mapping(self):
 
     def test_frames(self):
 
-        view = self.report.create_view({1,2,3})
-        timestamps, frames = view.load(0.0,0.2)
+        view = self.report.create_view({1, 2, 3})
+        timestamps, frames = view.load(0.0, 0.2)
 
         assert(numpy.isclose(timestamps,[ 0.0, 0.1]).all())
         assert(numpy.isclose(frames, [[-65., -65., -65.],
             [-65.14350891, -65.29447937, -65.44480133]]).all())
 
-        #### load(start,end)
-        timestamps, frames = view.load(0.05,0.25)
+        # load(start, end)
+        timestamps, frames = view.load(0.05, 0.25)
         # This window overlaps frames [0, 0.1), [0.1, 0.2), [0.2, 03)
         assert(len(timestamps) == 3)
-        assert(frames.shape == (3,3))
-        timestamps, frames = view.load(0.09,0.12)
+        assert(frames.shape == (3, 3))
+        timestamps, frames = view.load(0.09, 0.12)
         # Despite the time window is smaller than the timestep, each edge is
         # falling on a different frame, so we get two frames.
         assert(len(timestamps) == 2)
         assert(frames.shape == (2, 3))
 
-
-        ### load(start,end,step)
-        timestamps, frames = view.load(0.0,1.0,0.2)
+        # load(start, end, step)
+        timestamps, frames = view.load(0.0, 1.0, 0.2)
         assert(len(timestamps) == 5)
         assert(numpy.isclose(timestamps,[ 0.0, 0.2, 0.4, 0.6, 0.8]).all())
-        assert(frames.shape == (5,3))
+        assert(frames.shape == (5, 3))
+
+        timestamps, frames = view.load(0.0, 1.0, 0.3)
```

Comment:
The problem was not the accuracy of the test but its incompleteness. If was working only for cases there step = reportStep * n + epsilon. If step was 0.29999999... (0.3 in double) and reportStep 0.1, the test was failing because fmod was 0.0999999, that is reportStep - epsilon.